x-pack/plugins/reporting/server/routes/lib/request_handler.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
```

Comment:
This code was pulled out of `x-pack/plugins/reporting/server/routes/generation.ts` to allow better reusability